### DISCUSSION TITLE
Add config option to add back previous lazer progress bar

### DISCUSSION THREAD

**Discussion by SonoSooS** [ppy/osu#21381] [Ideas] [answered]
Since half a year a lot has changed in lazer, and it seems like the lazer progressbar has been got rid of in favor of the default pie of the proprietary client.

Is there a way to bring it back? I've scrolled through the settings multiple times, and I can't find any menu option capable of switching between the progress bars, just like you can on the proprietary client.

**Comment by Theighlin** [comment, marked as answer]
You can add it via the skin editor, press Ctrl+shift+s or use the butto. "open layout editor" in settings.

**Reply by SonoSooS** [reply]
I didn't find it under Gameplay, I checked that as well.

Also does it save for all skins?

**Reply by Theighlin** [reply]
No, it is saved per skin.

**Reply by SonoSooS** [reply]
That's unfortunate, it would be rather painful to set this for every skin I have.

**Reply by peppy** [reply]
There will eventually be better options to copy elements between skins.

**Reply by SonoSooS** [reply]
Will it be a global template skin (that is, it gets applied to every skin, unless I unlink a specific skin from the main template), or will I have to apply it manually to any new skin I import from the proprietary client?

**Reply by peppy** [reply]
We shall see.
